test/tests/functional/pbs_moved_job.py

```diff
@@ -114,3 +114,19 @@ def test_moved_job_history(self):
             qstat = ""
 
         self.assertEqual(qstat, "")
+
+    def test_movejob_to_unknown_host(self):
+        """
+        This test verifies the error message qmove gives when user tries to
+        move a job to an unknown server
+        """
+
+        attr = {'scheduling': 'false'}
+        self.server.manager(MGR_CMD_SET, SERVER, attr)
+
+        j = Job(TEST_USER)
+        jid = self.server.submit(j)
+
+        err = "Access from host not allowed, or unknown host " + jid
+        with self.assertRaises(PbsMoveError, msg=err):
```

Comment:
msg is the message printed in the PTL log when this exception is not raised.  I think you want to print something like 'qmove not rejected'.  You want to say 'with self.assertRaises() as e:' and then after after your with statement, you can say self.assertEqueals(e.exception.msg, err)

Reply:
ahh... my bad :)
I'll correct it :)